conference.js

```diff
@@ -2611,6 +2611,43 @@ export default {
         room.sendEndpointMessage(to, payload);
     },
 
+    /**
+     * Creates a new poll in the chat room.
+     * @param {Object} poll - Object containing the information about the poll.
+     * @param {Object} choices - Object containing poll choices by their ID.
+     * @param {Object} question - Object with information about
+     *  the question asked.
```

Comment:
One that thing's slightly confusing me here is the poll object contains information about the poll but it doesn't have all the information, considering you have to also pass in choices and the question. Maybe update the docs or shove everything into one object?

Reply:
To be honest its confusing especially with the way I documented it. I will update the docs for now, until we conclude the discussion about the structure of the state.